lib/ci.js

```diff
@@ -35,6 +37,27 @@ const ci = async () => {
   ])
   // npm ci should never modify the lockfile or package.json
   await arb.reify({ ...npm.flatOptions, save: false })
+
+  // run the same set of scripts that `npm install` runs.
+  const scripts = [
+    'preinstall',
+    'install',
+    'postinstall',
+    'prepublish', // XXX should we remove this finally??
```

Comment:
Just wondering, I thought `prepublish` was deprecated?

Reply:
please don't remove it yet :-) imo the churn isn't worth it. it'd be better to remove prepublishOnly and make `prepublish`, only running on prepublish, the actual behavior.

Reply:
I think since we already shipped v7, we have this on the agenda for npm v8.  It's a breaking change if we remove it, and at least an annoying change (if not actually breaking) if we deprecate it with a warning.

- remove (or deprecate with a warning) `prepublish` on install/ci
- remove (or deprecate with a warning) `prepublishOnly`

Reply:
Given that’s the plan, I’d prefer the former :-D (which, iirc, was the original planned end state)